pkg/test/utils/kubernetes.go

```diff
@@ -998,15 +999,24 @@ func Kubeconfig(cfg *rest.Config, w io.Writer) error {
 			})
 		}
 	}
-
+	var CAData []byte
+	var err error
+	if len(cfg.TLSClientConfig.KeyData) != 0 {
+		CAData = cfg.TLSClientConfig.CAData
```

Comment:
Can we use the LoadTLSFiles method instead? https://godoc.org/k8s.io/client-go/rest#LoadTLSFiles

Reply:
done